app/utils/workflow.js

```diff
@@ -46,9 +46,9 @@ export default class WorkflowUtil {
     // we don't count the 3 periods towards the max length (meaning, our resulting label will
     // technically go over the max length we detect).
     // eslint-disable-next-line prettier/prettier
-    const beginningPart = name.substring(0, (Constants.MAX_GRAPH_LABEL_LENGTH / 2));
+    const beginningPart = name.substring(0, Constants.MAX_GRAPH_LABEL_LENGTH / 2);
     // eslint-disable-next-line prettier/prettier
-    const endPart = name.substring(name.length - (Constants.MAX_GRAPH_LABEL_LENGTH / 2) + 1);
+    const endPart = name.substring(name.length - Constants.MAX_GRAPH_LABEL_LENGTH / 2 + 1);
```

Comment:
I'd prefer not to use the linted code - this has an eslint-disable command that was added to explicitly avoid the linter suggestion.  This is a personal preference, but I feel strongly for this line of code the explicit parentheses should be included.